src/layout/layout.rs

```diff
@@ -1099,37 +1081,19 @@ mod tests {
     }
 
     #[test]
-    fn custom_cache_size() {
-        assert!(Layout::init_cache(10));
-        assert!(!Layout::init_cache(15));
+    fn cache_size() {
         LAYOUT_CACHE.with(|c| {
-            assert_eq!(c.get().unwrap().borrow().cap().get(), 10);
+            assert_eq!(c.borrow().cap().get(), Layout::DEFAULT_CACHE_SIZE);
         });
-    }
 
-    #[test]
-    fn default_cache_size() {
-        let target = Rect {
-            x: 2,
-            y: 2,
-            width: 10,
-            height: 10,
-        };
+        Layout::init_cache(10);
+        LAYOUT_CACHE.with(|c| {
+            assert_eq!(c.borrow().cap().get(), 10);
+        });
 
-        Layout::default()
-            .direction(Direction::Vertical)
-            .constraints([
-                Constraint::Percentage(10),
-                Constraint::Max(5),
-                Constraint::Min(1),
-            ])
-            .split(target);
-        assert!(!Layout::init_cache(15));
+        Layout::init_cache(15);
         LAYOUT_CACHE.with(|c| {
-            assert_eq!(
-                c.get().unwrap().borrow().cap().get(),
-                Layout::DEFAULT_CACHE_SIZE
-            );
+            assert_eq!(c.borrow().cap().get(), 15);
```

Comment:
Probably don't need to test this twice now - this test was to show that it didn't change on the second call. Now that it does, this seems unnecessary.